digitalpublicgoods/opensrp.json

```diff
@@ -0,0 +1,88 @@
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://smartregister.atlassian.net/wiki/spaces/Documentation/overview"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "We have Beam connectors that can do this."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "We have measures to support GDPR compliance in place but none of the privacy laws listed affect our implementations yet.  We typically host or applications in country to comply with local privacy and data sovereignty laws."
+    ],
+    "adherenceSteps": [
+      "https://company.ona.io/ona-news/"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "FHIR.  We are also building a next generation version of OpenSRP that will be FHIR native."
+    ],
+    "evidenceStandardSupport": [
+      ""
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "We develop with an agile methodology.  We also try to provide test coverage for all aspects of the app.  We also work to ensure that all aspects of the technology do not rely on technologies that can not be run locally in country. "
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "We try to ensure our health apps always follow WHO approved health guidelines. We work with our implementing partners to help ensure this."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "Health information on people"
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "We encrypt and follow best security practices to secure the database where the PII is stored.  When doing analytics we endeavor to extract any PII data before it is stored in the data warehouse for analysis."
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "No",
+      "addressSafetySecurityUnderageUsers": "",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [],
+      "griefAbuseHarassmentProtection": "",
+      "harassmentProtectionSteps": [
+        ""
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
```

Comment:
Though not required, it would be really nice to have countries listed here. Thoughts? @nathanbaleeta

Reply:
Resolved in commit 3e119f4